Documentation/PageTsconfig/TcaDefaults.rst

```diff
@@ -33,3 +33,7 @@ Example:
 
    # Show newly created pages by default
    TCAdefaults.pages.hidden = 0
+
+..  note::
+    `TCAdefaults` is not (yet) applied to FlexForm values. These can only be addressed via
```

Comment:
```suggestion
    `TCAdefaults` is not applied to :ref:`FlexForm <t3coreapi:flexforms>` values. These can only be addressed via
```

I am a big fan to reference other chapters ;-), especially, when a word is dropped which is not self-explanatory.
Additionally, I would remove "(yet)" as this wakes expectations which might not be fulfilled in the near future.